main.qml

```diff
@@ -36,17 +36,22 @@ Window {
             right: parent.right
             margins: 10
         }
+        visible: false // Initially hidden
     }
 
     DroneTrackingPanel {
         id: droneTrackingPanel
-        anchors {
-            top: parent.top
-            left: parent.left
-            margins: 10
+        // Signal connection: when a drone is clicked, show the status panel
+        onUpdateSelectedDroneSignal: {
```

Comment:
![Screenshot 2024-10-26 110004](https://github.com/user-attachments/assets/dec498a4-b1e3-46f7-87dd-f2bf5304a2d3)
I think this isn't actual error, but more because its fake data currently, not tims fault i think